src/Php74/Php74.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php74;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php74
+{
+    /**
+     * @param object $obj
+     *
+     * @return array
+     */
+    public static function get_mangled_object_vars($obj)
+    {
+        if (!\is_object($obj)) {
+            trigger_error('get_mangled_object_vars() expects parameter 1 to be object, '.\gettype($obj).' given', E_USER_WARNING);
+
+            return null;
+        }
+
+        if ($obj instanceof \ArrayIterator || $obj instanceof \ArrayObject) {
+            $reflector = new \ReflectionClass($obj instanceof \ArrayIterator ? 'ArrayIterator' : 'ArrayObject');
+            $flags = $reflector->getMethod('getFlags')->invoke($obj);
+            $reflector = $reflector->getMethod('setFlags');
+
+            $reflector->invoke($obj, ($flags & \ArrayObject::STD_PROP_LIST) ? 0 : \ArrayObject::STD_PROP_LIST);
+            $arr = (array) $obj;
+            $reflector->invoke($obj, $flags);
+        } else {
+            $arr = (array) $obj;
+        }
+
+        return array_combine(array_keys($arr), array_values($arr));
+    }
+
+    /**
+     * @param string      $string
+     * @param int         $split_length
+     * @param string|null $encoding
+     *
+     * @return array|false
+     */
+    public static function mb_str_split($string, $split_length = 1, $encoding = null)
+    {
+        if (null === $encoding) {
+            $encoding = mb_internal_encoding();
+        }
+
+        if ($split_length < 1) {
+            trigger_error('The length of each segment must be greater than zero', E_USER_WARNING);
+
+            return false;
+        }
+
+        $result = array();
+        $length = mb_strlen($string, $encoding);
```

Comment:
Looks like we need `mb_` functions for this polyfill, how about adding `symfony/polyfill-mb` as a direct dependency? Otherwise this could break on certain systems.

Or maybe the mb polyfill should require this one, so that it polyfills all the `mb_` functions possible, i'm not sure what the standard is for this project.

Reply:
I was also wondering about that. Seems that the code should be duplicated - `mb_chr` is in both mbstring and PHP 7.2 polyfills.

Reply:
Added `symfony/polyfill-mbstring` as a dependency and duplicated the code following `IDN` and `mb_chr` polyfills.